scripts/release.sh

```diff
@@ -0,0 +1,252 @@
+submodule_current_version=$(jq -n "{$submodule_info}" | jq -r .from)
+
+# figure out latest version (visit link?)
+submodule_latest_version=$(gh repo view $submodule_url --json latestRelease -q .latestRelease.tagName)
+
+# compare versions
+if [ $submodule_current_version != $submodule_latest_version ]; then
+    echo
+    echo "Current version of $submodule_name: "$submodule_current_version
+    echo "Latest version of $submodule_name: "$submodule_latest_version
+    echo "Please update the submodule $submodule_name before continuing with the release."
+    echo
+    echo "You must also edit the dependancy information in SmartDeviceLink-iOS.podspec"
+    read user_input
+fi
+
+# 5 update changelog
+# TODO - insert a template into the changelog that includes the version the users have selected above.
+#echo "A template for this release has been inserted into the changelog.  Please update it."
+echo 
+echo "Please update Changelog.md, then return here and press enter."
+read user_input
+# Changelog.md
+
+# 6 generate documentation
+echo 
+echo "Would you like to automatically generate documentation with Jazzy (Y/n)?"
+read user_input
+if [[ ! $user_input == [Nn] ]]; then    
+    # 6 Install Jazzy
+    # Check if Jazzy is already installed, and if not then install jazzy
+    if [ -z "$(mdfind -name 'Jazzy')" ]; then
+        echo "Jazzy is not installed, attempting to install (may require your password)..."
+        sudo gem install jazzy
+    fi
+
+    # this runs Jazzy to generate the documentation
+    echo "Running Jazzy to generate documentation..."
+    jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+fi
+
+# 7 Ensure that the RPC_SPEC has released to the master branch and update the submodule to point to the new release tag (or to the HEAD of master, if no release of the RPC_SPEC is occurring).
+echo 
+echo "The rpc spec site will be opened for you"
+echo "https://github.com/smartdevicelink/rpc_spec"
+echo "Please check if there is a new release of the RPC_SPEC to master."
+echo "If there is, please update the rpc_spec submodule to point to the new master, then press enter."
+read user_input
+open "https://github.com/smartdevicelink/rpc_spec"
+read user_input
+#TODO - phase ? - can this be automated.  Check version.  Check version at site.
+
+echo
+echo "Please perform the following steps to push the release to master:"
+# and every step of this process, the user needs complete control.
+# that means we tell them what we want to do, ask them if they want us to do it, report the results, and gracefully handle exits.
+
+# git commands
+# 8 commit release to develop
+echo "Would you like to commit these changes to the develop branch? Y/n"
+read user_input
+#if [[ ! $user_input == [Nn] ]]; then
+#todo - phase 3 - run the commands automatically
+echo "Please Commit and push the changes to develop"
```

Comment:
You're good to add the git parts in now. Sorry if that was confusing last time, that's what I was trying to point for you to do.